modes/vterm/evil-collection-vterm.el

```diff
@@ -62,6 +62,58 @@ also uses `evil-mode'."
                        "vterm"
                      "emacs"))))
 
+(defun vterm-evil-insert ()
+  (interactive)
+  (vterm-goto-char (point))
+  (call-interactively #'evil-insert))
+
+(defun vterm-evil-insert-line ()
```

Comment:
Having a hard time grokking what this does (or what someone would use it for).

Reply:
This is the equivalent to the I motion map in vim for going back to the beginning of the (editable) portion of the prompt. It's a little finnicky due to prompt detection with vterm (working on it), but I use it for changing the first command that I've entered.

Reply:
Added documentation :)